README.md

```diff
@@ -7,7 +7,74 @@
+
+```text
+<Question 1> → <model’s answer 1> → <Follow-up question> → <model’s answer 2>
+```
+
+and having a “judge” model review the given multi-turn question, the provided model answer, and rate the answer with a score out of 10. The scores are then averaged out
+and the final score produced is the “MT-bench score” for that model. This benchmark assumes no factual knowledge on the model’s part. The questions are static, but do
+not get obsolete with time.
+
+You can read more about MT-Bench [here](https://arxiv.org/abs/2306.05685)
+
+### MT-Bench Branch
+
+MT-Bench Branch is an adaptation of MT-Bench that is designed to test custom skills that are added/being added to the model via the InstructLab project. These new skills
+come in the form of question/answer pairs in a Git branch of the [taxonomy](https://github.com/instructlab/taxonomy).
+
+MT-Bench Branch uses the user supplied seed questions to have the candidate model generate answers to, which are then judged by the judge model.
+
+### Massive Multitask Language Understanding (MMLU)
+
+**tl;dr** Full model evaluation of performance on **knowledge**
+
+MMLU is a type of benchmarking that involves a series of fact-based multiple choice questions, along with 4 options for answers. It tests if a model is able to interpret
+the questions correctly, along the answers, formulate its own answer and pick the correct option out of the provided ones. The questions are designed as a set of 57
+“tasks”, and each task has a given domain. The domains cover a number of topics ranging from Chemistry and Biology to US History and Math.
+
+The performance is then compared against the set of known correct answers for each question to determine how many the model got right. The final MMLU score is the
+average of its scores. This benchmark does not involve any reference/critic model, and is a completely objective benchmark. This benchmark does assume factual knowledge
+on the model’s part. The questions are static, therefore MMLU cannot be used to gauge the model’s knowledge on more recent topics.
+
+InstructLab uses an implementation found [here](https://github.com/EleutherAI/lm-evaluation-harness) for running MMLU.
+
+You can read more about MMLU [here](https://arxiv.org/abs/2306.05685)
+
+### MMLU Branch
+
+MMLU Branch is an adaptation of MMLU that is designed to test custom knowledge that is being added to the model via a Git branch of the [taxonomy](https://github.com/instructlab/taxonomy).
+
+A teacher model is used to generate new multiple choice questions based on the knowledge document included in the taxonomy Git branch. A “task” is then constructed that references the newly generated questions and choices. These tasks are then used to judge the model’s grasp on new knowledge the same way MMLU works. Generation of these tasks are done as part of the [InstructLab SDG](https://github.com/instructlab/sdg) library.
```

Comment:
Would "score the model's grasp" be more descriptive than judge in this case?

Reply:
Yes I think so! Updating to that